src/Microsoft.ML.StandardLearners/Properties/AssemblyInfo.cs

```diff
@@ -9,6 +9,7 @@
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.StaticPipe" + PublicKey.Value)]
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Core.Tests" + PublicKey.TestValue)]
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Predictor.Tests" + PublicKey.TestValue)]
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Tests" + PublicKey.TestValue)]
```

Comment:
I think @artidoro in his #2846 actually skipped this test, so we might be able to avoid this by simply removing the test. NOt sure what he thinks.

Reply:
Give what you highlighted I think we should remove the test, we already have a test with `TestEstimatorCore`. If we want to make this estimator public, I think we should write tests in the new API anyway. 

You can remove it here, or I can do it in my PR.